source/module_hamilt_pw/hamilt_ofdft/ml_tools/train.h

```diff
@@ -0,0 +1,89 @@
+#ifndef TRAIN_H
```

Comment:
this title is also too common, how about TRAIN_KEDF?

Reply:
OK.